HULKs/hulk
# Allow for parallel camera access

## Introduced Changes

Makes the camera access non-mutable so that it can be used from multiple cyclers.
This will be required for other cyclers that want to operate on camera images.

Fixes #

## ToDo / Known Issues
* [x] Test on NAO

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test
* upload check if everything still works
* check that images in twix are still displayed correctly
